src/main/java/com/training/web/pages/flipkart/FlipkartHomePage.java

```diff
@@ -9,9 +9,9 @@
     private WebDriver driver;
     private WebDriverWait wait;
 
-    private By searchBox = By.xpath("//*[@class='_3704LK']");
+    private By searchBox = By.xpath("//input[contains(@title,'Search for products')]");
     private By popup = By.xpath("/html/body/div[2]/div/div/button");
```

Comment:
for popup, lets use a more optimised xpath/locator. This one looks flaky and can break if the page structure changes